trade-finance-manager-api/src/v1/controllers/send-feedback-email.js

```diff
@@ -0,0 +1,21 @@
+const { NotifyClient } = require('notifications-node-client');
+require('dotenv').config();
+
+const notifyClient = new NotifyClient(process.env.GOV_NOTIFY_API_KEY);
+
+const sendEmail = async (templateId, sendToEmailAddress, emailVariables) => {
+  const personalisation = emailVariables;
+
+  await notifyClient
+    .sendEmail(templateId, sendToEmailAddress, {
+      personalisation,
+      reference: null,
+    })
+    .then((response) => response)
+    .catch((err) => {
+      console.error(`TFM API - Failed to send email to address: ${sendToEmailAddress} with templateId ${templateId}`);
+      return err.response;
+    });
+};
+
```

Comment:
Can we use `send-tfm-email` instead of duplicating the code and adding another package. All TFM emails use `sendTfmEmail` function which then calls reference data API.

Reply:
sendTfmEmail does not work as there is no deal for feedback. I can dupe the code for send-tfm-email and remove the deal part?

Reply:
done